lib/SmileIdentityCore.php

```diff
@@ -0,0 +1,289 @@
+<?php
+spl_autoload_register(function($class) {
+     require_once($class.'.php');
+});
+
+use \GuzzleHttp\Client;
+use \GuzzleHttp\Exception\GuzzleException;
+use \Psr\Http\Message\ResponseInterface;
+
+const VERSION = '1.1.0';
+const DEFAULT_JOB_STATUS_SLEEP = 2;
+
+class SmileIdentityCore
+{
+    const SID_SERVERS = [
+        'https://3eydmgh10d.execute-api.us-west-2.amazonaws.com/test',
+        'https://la7am6gdm8.execute-api.us-west-2.amazonaws.com/prod'
```

Comment:
@osa-si the SmileIdentityCore still has the old URLs for the server. IdApi has the new ones...maybe pull these out to one common place?